Notifier/UI/Windows/NotificationWindow.xaml.cs

```diff
@@ -402,13 +402,13 @@ private void btnOptions_Click(object sender, RoutedEventArgs e)
 
         public void ShowConsole()
         {
-            if (Process.GetProcessesByName(WFN_PROCESS_NAME).Length == 0)
+            if (Process.GetProcessesByName(ProcessHelper.WFNProcessEnum.WFN.ToString()).Length == 0)

```

Comment:
A constant would be better in fact. Enums are not meant to be used this way :-)

Reply:
But they are handy and the param is explicitly defined. The alternative is to create a class containing the process-names as constants if you want to be explicit what a "process name" is (is it "wfn.exe" as shown in the taskmanager or "wfn"?) - unless in .net has a better approch? 🤔

Reply:
I'm not sure I get it :-) The param would also be explicitly defined. If you want the name to match its value (to avoid having, for instance, a const named WFN_PROCESS_NAME which in fact contains something else, like "console" as it was at first), you can use a little trick, with public const string WFN = nameof(WFN);
That way, refactoring and renaming the const will also update it's own value (avoiding the need to use "nameof(WFN)" everywhere you need the process name.
You'd then be able to keep both WFN and Notifier in a sub class (to mimick an enum). In fact it would work the exact same way than you did, except you wouldn't have to call ToString since you already have the corresponding value.

Reply:
Ohh, I see now your confusion when you look only at this line. It was primarely intended to use it like this:

>Declaration: StartOrRestoreToForeground(WFNProcessEnum processName) 
Use: ProcessHelper.StartOrRestoreToForeground(ProcessHelper.WFNProcessEnum.WFN);

But, I just noticed that I forgot to remove the previously used constants WFN_EXE and WFN_PROCESS_NAME in the same class. No time to correct it at the moment though 😦

... and thanks for the hint with *nameof* 👍  - I agree that a subclass with constants would serve a similar purpose. Maybe we should also introcude a common GlobalConstants class  containing them - think about it when doing the refactoring 😃